src/Community.VisualStudio.Toolkit.Shared/Commands/BaseCommand.cs

```diff
@@ -96,7 +96,7 @@ protected virtual void Execute(object sender, EventArgs e)
                 {
                     await ex.LogAsync();
                 }
-            });
+            }).FireAndForget();
```

Comment:
Removing the null-conditional operator on this statement resulted in VSTHRD110 (_Observe the awaitable result of this method call by awaiting it, assigning to a variable, or passing it to another method._), hence why I've added the `FireAndForget()` call.